Fix: handling case where RL is lower then expected

tszumski opened the pull request:
This Pull request is based on and replace https://github.com/OpenVisualCloud/Media-Transport-Library/pull/1153

1. Fix handling case where RL is lower then expected
2. Update ice driver to version 2.2.8

Sakoram commented on lib/src/st2110/st_tx_video_session.c:
```diff
@@ -423,10 +425,25 @@ static int tv_train_pacing(struct mtl_main_impl* impl, struct st_tx_video_sessio
+      err("%s(%d), measured speed is too low on already trained bps\n", __func__, idx);
+      return -EINVAL;
+    }
+#define INCREASE_BPS_FACTOR 1.005
```
please add a comment why INCREASE_BPS_FACTOR  is needed.

tszumski replied:
Fixed in 87490591608abf90b86cf59a793c5c47e13952ca

tszumski commented:
> please update: https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/versions.env#L2 if you can check if this works, because $ICE_REPO seems to be broken. ICE_DMID should be redefined as ICE_DOWNLOAD_MIRROR and updated with your version change.

Fixed in 87490591608abf90b86cf59a793c5c47e13952ca

Sakoram reviewed (approved):
Please make sure that the commit message describes what was changed.
